accounts/abi/bind/dep_tree_test.go

```diff
@@ -305,5 +317,14 @@ func TestContractLinking(t *testing.T) {
 			'i': {},
 		},
 		map[rune]struct{}{
-			'a': {}, 'b': {}, 'c': {}, 'd': {}, 'e': {}, 'f': {}, 'g': {}, 'h': {}, 'm': {}}})
+			'a': {}, 'b': {}, 'c': {}, 'd': {}, 'e': {}, 'f': {}, 'g': {}, 'h': {}, 'm': {}},
+	},
+}
+
+func TestContractLinking(t *testing.T) {
+	for i, tc := range linkTestCases {
+		if err := testLinkCase(tc); err != nil {
+			t.Fatalf("test case %d failed: %v", i, err)
+		}
+	}
```

Comment:
How we typically do it is 

```golang

for i, tc := range []linkTestCaseInput{
// the whole list
}{
    // exeute the test
}
```
Or is there some point to having `linkTestCases` defined as a variable? 

```

Reply:
No reason to define it as a package-scoped variable.